src/components/HeaderWithMenu.tsx

```diff
@@ -22,25 +26,44 @@ const SmileIconStyle = styled.Pressable`
   top: 20px;
 `
 
-const MenuIconContainer = styled.Pressable`
+const HeaderButtonsContainer = styled.View`
```

Comment:
you can just use flexbox instead of positioning absolute

```js
const HeaderButtonsContainer = styled.View`
  align-self: flex-end;
`
```

Reply:
nice